modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -48,7 +48,7 @@ Dear ownCloud administrator, find below the changes and known issues in ownCloud
 When upgrading from ownCloud Server 10.11.0, the following migrations will run as part of the upgrade procedure:
 
 * A migration step takes care of setting the length of the "oc_calendars.components" column to 255 (see below). This step is expected to be quick and should not impact upgrade duration significantly. https://github.com/owncloud/core/pull/40563[#40563]
-* An index is going to be added on the oc_filecache table which should help to speed-up bulk file operations. https://github.com/owncloud/core/pull/40633[#40633]
+* An index is going to be added on the oc_filecache table which should help to speed-up bulk file operations. Plase be aware that this migration step may take several hours in case of installations where the oc_filecache table contains a significant large amount of entries. https://github.com/owncloud/core/pull/40633[#40633]
```

Comment:
```suggestion
* An index is going to be added on the oc_filecache table which should help to speed-up bulk file operations. Please be aware that this migration step may take several hours in case of installations where the oc_filecache table contains a significantly large number of entries. https://github.com/owncloud/core/pull/40633[#40633]
```

Reply:
thanks for noticing it